package.json

```diff
@@ -48,6 +48,7 @@
     "@tailwindcss/typography": "^0.5.8",
     "autoprefixer": "^10.4.13",
     "prettier": "^1.16.4",
+    "remark-mdx": "^2.0.0-next.9",
```

Comment:
I don't see you using this anywhere.

Reply:
Had to install it to get rid of some error while configuring the project in my local.